app/styles/gavant-ember-table.scss

```diff
@@ -0,0 +1,220 @@
+@import 'gavant-ember-table/variables';
+@import 'ember-bootstrap/bootstrap';
+@import './progress-spinner/progress-spinner';
+
+.data-table {
+    position: relative;
+    padding-bottom: $gavant-padding-bottom;
```

Comment:
i'd probably maybe these vars all start with the `$gavant-table-...` prefix so they are a bit more specific to what addon the are coming from (though `$gavant-table-` is a bit long for a prefix.. maybe we can shorten it to `$gav-tbl-` or something :-P)